airbyte-integrations/connectors/destination-s3/src/test/java/io/airbyte/integrations/destination/s3/avro/JsonSchemaTypeProvider.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.s3.avro;
+
+import java.util.stream.Stream;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.ArgumentsProvider;
+
+public class JsonSchemaTypeProvider implements ArgumentsProvider {
```

Comment:
Nitpick. Since this test case provider is only used in `JsonSchemaTypeTest`, it can be moved into the `JsonSchemaTypeTest.java` file as a static internal class. This makes the type test file more self-contained.

Reply:
Moved as static member, thanks.